cypress/support/commands.js

```diff
@@ -82,3 +82,15 @@ Cypress.Commands.add('typeCmdOrCtrl', () => {
 function isMac() {
     return navigator.platform.toUpperCase().indexOf('MAC') >= 0;
 }
+
+// ***********************************************************
+// Post
+// ***********************************************************
+
+Cypress.Commands.add('post', (message) => {
```

Comment:
```suggestion
Cypress.Commands.add('postMessage', (message) => {
```

Please change to `postMessage` 'cause in the future we might have `postComment` command.